requirements/openvino.txt

```diff
@@ -1,10 +1,10 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # OpenVINO Requirements.                                                      #
-# FIXME: nncf 2.3.0 has scipy version conflict
-# this is a temporary workaround to udpate nncf dependencies
-nncf @ git+https://github.com/cih9088/nncf@9b204857c11bc694c2ca6a62fc9a575c002a0a99#egg=nncf
+nncf==2.4.0
 onnx==1.10.1
 openmodelzoo-modelapi==2022.3.0
 openvino==2022.3.0
 openvino-dev[tensorflow2,mxnet,caffe]==2022.3.0
-mmdeploy @ git+https://git@github.com/open-mmlab/mmdeploy@v0.10.0
+# FIXME: Below is the only one left as git+https install
+# But, the latest one on PyPI is 0.2.0. Might need custom build & upload
+mmdeploy @ git+https://git@github.com/open-mmlab/mmdeploy@v0.12.0
```

Comment:
Would it be an idea to contact the mm folks for this? Like create an issue on their repo? Or email them directly?

Reply:
Yeah, I also thought so now :) I'll contact them.

Reply:
https://github.com/open-mmlab/mmdeploy/issues/1720